File: python/Day_1/Question_14.py
lower = int(input('enter lower limit: '))
upper = int(input('enter upper limit: '))

for number in range(lower, upper + 1):
    if(number > 1):
        for i in range(2, number):
            if(number % i == 0):
                break
        else:
            print(number)

File: dsa/Day_2/Input_Output/Question_14.cpp
#include<iostream>
using namespace std;

int main(){
    int num;

    cout << "enter number: ";
    cin >> num;

    for (int i = 0; i <= 10 ; i++){
        cout << num << " * " << i << " = " << num * i << endl;
    }

    return 0;
}